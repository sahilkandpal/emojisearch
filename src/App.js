import Header from "./components/Header/Header";
import SearchBar from "./components/SearchBar/SearchBar";
import EmojiList from "./components/EmojiList/EmojiList";
import Toast from "./components/Toast/Toast";
import { EmojiContextProvider } from "./context/emojiContext";
import { ToastContextProvider } from "./context/toastContext";
const App = () => {
  return (
    <div className="App">
      <Header />
      <EmojiContextProvider>
        <SearchBar />
        <ToastContextProvider>
          <EmojiList />
          <Toast />
        </ToastContextProvider>
      </EmojiContextProvider>
    </div>
  );
};

export default App;
