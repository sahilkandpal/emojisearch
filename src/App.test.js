import { shallow } from "enzyme";
import App from "./App";
describe("App", () => {
  test("App should be rendered wihout breaking", () => {
    const tree = shallow(<App />);
    expect(tree).toMatchSnapshot();
  });
});
