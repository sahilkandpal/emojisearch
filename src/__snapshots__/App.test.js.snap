// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App should be rendered wihout breaking 1`] = `
<div
  className="App"
>
  <Header />
  <EmojiContextProvider>
    <SearchBar />
    <ToastContextProvider>
      <EmojiList />
      <Toast />
    </ToastContextProvider>
  </EmojiContextProvider>
</div>
`;
